src/test/mocks/setup.ts

```diff
@@ -18,3 +18,7 @@ precreateApiInstance();
 if (!process.env?.REACT_APP_RELAY_CHAIN_NAME) {
   throw new Error('Please make sure you have created .env.test file with the necessary env variables');
 }
+
+// RE: inspired by https://stackoverflow.com/a/49864436
+// MEMO: the root cause of this fix may be that these operations (running tests) can take longer to complete in a CI environment than on your local machine so the timeout might be increased accordingly
+jest.setTimeout(30000);
```

Comment:
There is a jest config for this: `testTimeout`

Reply:
@danielsimao 
Sorry, I cannot find `testTimeout` in the code base.

Reply:
Done!